ti/report/parser/junit/gojunit/ingest.go

```diff
@@ -16,33 +16,40 @@ import (
 	ti "github.com/harness/ti-client/types"
 )
 
+const cypressRootSuiteName = "Root Suite"
+
 // findSuites performs a depth-first search through the XML document, and
 // attempts to ingest any "testsuite" tags that are encountered.
-func findSuites(nodes []xmlNode, suites chan Suite) {
+func findSuites(nodes []xmlNode, suites chan Suite, parentFilename string) {
 	for _, node := range nodes {
 		switch node.XMLName.Local {
 		case "testsuite":
-			suites <- ingestSuite(node)
+			suites <- ingestSuite(node, parentFilename)
+			if node.Attr("name") == cypressRootSuiteName {
+				parentFilename = node.Attr("file")
+			}
 		default:
-			findSuites(node.Nodes, suites)
+			findSuites(node.Nodes, suites, parentFilename)
 		}
 	}
 }
 
-func ingestSuite(root xmlNode) Suite { //nolint:gocritic
+func ingestSuite(root xmlNode, parentFilename string) Suite { //nolint:gocritic
 	suite := Suite{
 		Name:       root.Attr("name"),
 		Package:    root.Attr("package"),
 		Properties: root.Attrs,
 	}
 
+	file := getFilename(root.Attr("file"), parentFilename)
```

Comment:
Why should this be changed to parentFilename? It should be file right since this is what we will now consider as the file for this Test Suite

Reply:
It's just naming convention. It's technically a `parent` since you're passing it to `ingestSuite`. You can just re-use the same variable name since it's a recursion.

Reply:
Done